### ISSUE TITLE
force-glow will apply depth-strider to enchanted books in the new AE version

### ISSUE THREAD

**Issue by xlsalvolx** [GC-spigot/AdvancedEnchantments#4708]
### Describe the bug

```
enchantment-book:
  # If true, books can be applied to items in anvils.
  anvil-application: false
  # If true, books can be applied to items by dragging and dropping them on the item in an inventory.
  drag-drop-application: true
  item:
    type: KNOWLEDGE_BOOK
    id: 0
    custom-model-data: 0
    force-glow: true
  # Variables: %enchant%, %enchant-no-color%, %description%, %level-description%, %applies%, %success%, %destroy%, %color%, %max-level%
  name: '%group-color%&l&n%enchant-no-color% %level%'
  lore:
  - '&a%success%% Successo'
  - '&c%destroy%% Fallimento'
  - '&e%description%'
  - '&7Utilizza /enchants per sapere a quali'
  - '&7Item puoi applicare questo incantesimo'
  - '&8Trascina su un item per applicare'
```

Setting "force-glow" on true will now apply depth-strider to enchanted books, rendering every enchanted book prior to this update outdated and unusable (and also the depth-stider is really ugly)

### How to reproduce

1. Turn force-glow to true
2. Give yourself a book
3. Done

### Screenshots / Videos

![2024-11-01_16 02 00](https://github.com/user-attachments/assets/e53e204b-b8f8-4c10-8680-10195274fbfd)

### "/ae plinfo" link

https://paste.md-5.net/alopacujov

### Filled out form correct and using latest version

👌

**Comment by GC-spigot** [comment]
We've updated the code but problem is with versions before 1.20.5 not having a real way to apply enchantment glow, so from 1.20.6 all versions fixes it, before then this was the way to do it.

**Comment by xlsalvolx** [comment]
so is the only way to fix this updating to 1.20.5? or is this going to get fixed for older versions?

**Comment by GC-spigot** [comment]
I've updated the code. Enchantments still get applied but also a hide enchants itemflag does as well. Should be better.
